packages/mui-material/src/ButtonGroup/ButtonGroupButtonContext.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+interface IButtonGroupButtonContext {
```

Comment:
We don't prefix interfaces with "I"

```suggestion
interface ButtonGroupButtonContextType {
```